README.md

```diff
@@ -15,11 +15,11 @@ Obtain a copy of Hive from GitHub at https://github.com/apache/hive.
 
 To build the Hive client, you need to first apply this [patch](https://issues.apache.org/jira/secure/attachment/12958418/HIVE-12679.branch-2.3.patch).  Download this patch and move it to your local Hive git repository you created above.  Apply the patch and build Hive.
 
-	git checkout branch-2.3
+	git checkout tags/rel/release-2.3.4 -branch rel-2.3.4
```

Comment:
Should it be "-b"?
`git checkout tags/rel/release-2.3.4 -b rel-2.3.4`

Note: I am running git version 2.17.2 (Apple Git-113)

Reply:
Yes, typo on my part. Thanks for the correction.